src/Whoops/Handler/PrettyPageHandler.php

```diff
@@ -95,6 +95,7 @@ class PrettyPageHandler extends Handler
         "macvim"   => "mvim://open/?url=file://%file&line=%line",
         "phpstorm" => "phpstorm://open?file=%file&line=%line",
         "idea"     => "idea://open?file=%file&line=%line",
+        'vscode'   => "vscode://file/%file:%line",
```

Comment:
Please use consistent quote style with the code around. In this case the `vscode` part should use double quotes.